Libraries/TestHelpers.psm1

```diff
@@ -368,7 +368,8 @@ Function Run-LinuxCmd([string] $username,[string] $password,[string] $ip,[string
 			$logCommand = "`"$MaskedCommand`""
 		}
 	}
-	Write-LogInfo ".\Tools\plink.exe -t -pw $password -P $port $username@$ip $logCommand"
+	$output = Write-Output "yes" | & ".\Tools\plink.exe" -t -pw $password -P $port $username@$ip $logCommand
```

Comment:
Even if you want to do echo yes, this is not the correct place.
This is just doing Write-LogInfo. The plink.exe is not getting invoked here.